pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -60,6 +60,10 @@ type RunSpec struct {
 	// +optional
 	Status RunSpecStatus `json:"status,omitempty"`
 
+	// Status message for cancellation.
+	// +optional
+	StatusMessage RunSpecStatusMessage `json:"status_msg,omitempty"`
```

Comment:
The json tag should be `statusMessage` for consistency.

Reply:
Done.